packages/stories-block/src/block/block-types/single-story/edit.js

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+/* eslint complexity: ["error", { "max": 21 }] */
```

Comment:
Sounds like it's time for some refactoring soon to fix this instead of just bumping the number here